src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -352,34 +364,39 @@ def open_pr_comment_workflow(pr_id: int) -> None:
     # CREATING THE COMMENT
 
     # fetch the files in the PR and determine if it is safe to comment
-    safe_to_comment, pr_files = safe_for_comment(
-        gh_client=client, repository=repo, pull_request=pull_request
-    )
+    pr_files = safe_for_comment(gh_client=client, repository=repo, pull_request=pull_request)
 
-    if not safe_to_comment:
-        logger.info("github.open_pr_comment.not_safe_for_comment")
+    if len(pr_files) == 0:
+        logger.info(
+            "github.open_pr_comment.not_safe_for_comment", extra={"file_count": len(pr_files)}
+        )
         metrics.incr(
             OPEN_PR_METRICS_BASE.format(key="error"),
             tags={"type": "unsafe_for_comment"},
         )
         return
 
-    # TODO(cathy): return patches
-    pr_filenames, _ = get_pr_filenames_and_patches(pr_files)
+    pr_filenames, patches = get_pr_filenames_and_patches(pr_files)
```

Comment:
could also add a comment here saying we assume that the returned lists should be the same in size